_literate/EX-boston-flux/tutorial.jl

```diff
@@ -134,14 +133,13 @@ curve = MLJ.learning_curve(nnregressor, features, targets,
                        resampling=MLJ.Holdout(fraction_train=0.7),
                        measure=MLJ.l2)
 
-figure(figsize=(8,6))
+using Plots
 
-plt.plot(curve.parameter_values,
-    curve.measurements)
+plot(curve.parameter_values, curve.measurements, yaxis=:log, legend=false)
 
-yscale("log")
-xlabel(curve.parameter_name)
-ylabel("l2")
+#yscale("log")
```

Comment:
Can't we do `yscale=:log10` here? (If I remember, :log might not be supported, but I was pretty sure `:log10` was.)

Reply:
I have `yaxis:=log` as shown in this [SO question](https://stackoverflow.com/questions/51446080/how-to-make-a-log-plot-in-julia). The comment should be removed.